src/store/concepts/session/types.js

```diff
@@ -0,0 +1,6 @@
+const namespace = 'SESSION'
+
+export const SUBMIT_USER_LOGIN = `${namespace}/SUBMIT_USER_LOGIN`
```

Comment:
rename this type, action etc to `userLoginSubmit` or even `loginSubmit`. i think you can remove prefixes like `USER_` from all of actions

Reply:
Ok, thanks, good point